applications/ConstitutiveModelsApplication/tests/generalTests.py

```diff
@@ -0,0 +1,28 @@
+from KratosMultiphysics import *
+from KratosMultiphysics.ConstitutiveModelsApplication import *
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+
+def GetFilePath(fileName):
+    return os.path.dirname(__file__) + "/" + fileName
+
+
+class KratosConstitutiveModelsGeneralTests(KratosUnittest.TestCase):
```

Comment:
This test seems to do nothing. No need to include it just because its need to have one, If you intend to add tests latter it is ok.

Reply:
We are buiding the tests, they are not included there.